sql/src/main/java/io/crate/jobs/AbstractExecutionSubContext.java

```diff
@@ -41,6 +41,7 @@
 
     protected AbstractExecutionSubContext(int id) {
         this.id = id;
+        logger = Loggers.getLogger(this.getClass());
```

Comment:
Would it make sense to add a logger as argument to the constructor instead? `getLogger` can be fairly expensive